Documentation/docker_guide.md

```diff
@@ -12,9 +12,11 @@ export HostIP="192.168.12.50"
 
 The following `docker run` command will expose the etcd client API over ports 4001 and 2379, and expose the peer port over 2380.
 
+This will run the latest release version of etcd. You can specify version if needed (e.g. `quay.io/coreos/etcd:v*`).
```

Comment:
`quay.io/coreos/etcd:v2.2.0` instead of `quay.io/coreos/etcd:v*`? I think we are trying to give a specific example, right?